recipes/impute5/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: impute5
+  version: '1.1.5'
+
+source:
+  url: https://github.com/LouisLeNezet/Impute5/raw/main/impute5_v1.1.5.zip
+  sha256: 0386abc7061498267edc1d03cba5843fdcf33b9b9aa05fe06de8821270635f50
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  build:
+  run:
+
+test:
+  commands:
+    - 'impute5 --help'
+    - 'imp5Chunker --help'
+    - 'imp5Converter --help'
+about:
+  home: https://jmarchini.org/software/#impute-5
+  license: Academic use, commerically restricted (https://www.dropbox.com/sh/mwnceyhir8yze2j/AADbzP6QuAFPrj0Z9_I1RSmla?dl=0&preview=LICENCE.txt)
```

Comment:
While it does not look like the license is required to be distributed with the software, it would not be a bad idea to include it.